### PR TITLE
Departmental Security: The Fourth

### PR DESCRIPTION
**Pull request by Geevies**
Had a map conflict, checking if making a new PR fixes that.

Refer to this for pictures:
https://github.com/Aurorastation/Aurora.3/pull/7264

### CODE REVIEW

**Comment by Geevies** [comment]
holy fuck travis please

**Comment by furrycactus** [comment]
Two of the offices have command airlocks instead of security ones, and the research office has no windows. Otherwise, looks good. Ideally, make sure Supply officers don't have warehouse access, as well.

**Comment by Geevies** [comment]
!review

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -322,33 +320,49 @@
 		new /obj/item/clothing/accessory/holster/hip(src)
 		new /obj/item/weapon/storage/belt/security(src)
 
+/obj/structure/closet/secure_closet/security/brig
+	req_access = list(access_security, access_weapons)
+
+	fill()
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: do I see a relative pathed proc here

Comment (Geevies) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: im going to smack your ass

**Comment by Geevies** [comment]
Medical Office:
image

Engineering Office:
image

Research Office:
image

Cargo Office:
image

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -164,6 +164,35 @@
...
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the quartermaster, head of personnel, and head of security"
+	selection_color = "#adadad"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_maint_tunnels, access_mailsorting, access_cargo, access_qm, access_mining, access_mining_station)
+	minimal_access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_mining, access_mining_station, access_maint_tunnels, access_cargo, access_cargo_bot, access_mailsorting)
```
Comment (Arrow768) [inline code/game/jobs/job/civilian.dm]: `minimal_access` has more access than `access`

We are running on `minimal_access` so make sure that they are either the same or `access` has actually more access than `minimal_access`

File: code/game/jobs/job/engineering.dm [hunk 3]
```diff
@@ -148,3 +148,33 @@
...
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the chief engineer and head of security"
+	selection_color = "#dbba35"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_construction, access_atmospherics)
+	minimal_access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_construction, access_atmospherics,)
```
Comment (Arrow768) [inline code/game/jobs/job/engineering.dm]: `minimal_access` has more access than `access`
`minimal_access` has a trailing `,`

We are running on `minimal_access` so make sure that they are either the same or `access` has actually more access than `minimal_access`

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 4]
```diff
@@ -255,100 +255,114 @@
...
+	if (prob(50))
+		new /obj/item/clothing/accessory/storage/black_vest(src)
+	else
+		new /obj/item/clothing/accessory/storage/pouches/black(src)
+	new /obj/item/weapon/storage/belt/security(src)
 
 /obj/structure/closet/secure_closet/security
 	name = "security officer's locker"
-	req_access = list(access_brig)
+	req_access = list(access_weapons)
```
Comment (Arrow768) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Dont use `access_weapons` for that.
`access_weapons` is used to determine if someone is allowed to use weapons, not if they are a security officer.

File: code/game/machinery/computer/records.dm [hunk 5]
```diff
@@ -44,7 +44,7 @@
 
 	icon_screen = "security"
 	light_color = LIGHT_COLOR_ORANGE
-	req_one_access = list(access_security, access_forensics_lockers, access_lawyer, access_hop)
+	req_one_access = list(access_security, access_weapons, access_forensics_lockers, access_lawyer, access_hop)
```
Comment (Arrow768) [inline code/game/machinery/computer/records.dm]: Dont use `access_weapons` for that.
`access_weapons` is used to determine if someone is allowed to use weapons, not if they are a security officer.

**Pull request closed without merging**